### PR TITLE
Automotive Skill - EmbeddedResources and Docs update for Dispatch integration

### PR DESCRIPTION
**Pull request by darrenj**
## Description
- Moved the supporting setting JSON files/TSV files to embedded-resources so they are put directly into the assembly rather than content files that cause numerous deployment/path issues.
- Updated docs with steps to add skill to an existing VA

## Checklist
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have added or updated the appropriate unit tests
- [ x] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) property is set to enable a high quality speech-first experience and the appropriate [InputHints](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) are set correctly. 
- [x ] I have updated related documentation

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj [hunk 1]
```diff
@@ -6,6 +6,15 @@
     <CodeAnalysisRuleSet>..\..\..\VirtualAssistant.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Remove="Dialogs\VehicleSettings\Resources\available_settings.json" />
+    <Content Remove="Dialogs\VehicleSettings\Resources\setting_alternative_names.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Dialogs\VehicleSettings\Resources\normalization\amount_percentage.tsv" />
```
Comment (outofthecave) [inline solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj]: Is there a reason why `amount_type.tsv` and `amount_unit.tsv` are not listed here?

File: solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj [hunk 2]
```diff
@@ -72,15 +81,10 @@
...
-    <Content Include="Dialogs\VehicleSettings\Resources\normalization\amount_percentage.tsv">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Dialogs\VehicleSettings\Resources\normalization\amount_type.tsv">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Dialogs\VehicleSettings\Resources\normalization\amount_unit.tsv">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
+    <EmbeddedResource Include="Dialogs\VehicleSettings\Resources\available_settings.json" />
```
Comment (outofthecave) [inline solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj:75]: Is there a reason why `setting_alternative_names.json` is not listed here?

File: solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj (hunk 1 above)
Comment (darrenj) [reply solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj]: Good spot, VS adds these remove things randomly it seems. I'll remove as they aren't needed

File: solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj (hunk 2 above)
Comment (darrenj) [reply solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj:75]: The setting change in VS didn't seem to persist, great spot!

Comment (darrenj) [reply solutions/Virtual-Assistant/src/csharp/skills/automotiveskill/automotiveskill/AutomotiveSkill.csproj:75]: Actually it did but buried below (out of line). I'll collapse into one section for clarity

**Review by outofthecave — APPROVED** [review]

**Pull request merged**